.github/workflows/android-ci.yml

```diff
@@ -13,17 +13,22 @@ jobs:
   buildTest:
     name: Build JDK ${{ matrix.java_version }}
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
I think we should remove this line. Does it imply that a failure during this build would not cause the action to fail?

Reply:
@joelittlejohn Removing this line would halt the pipeline at the first sign of failure, which is not ideal as it prevents us from getting a full view of the entire process.

Reply:
Ah I see. So an error causes the build to be marked as a failure, but we don't stop immediately.

Reply:
@juherr Exactly